content/docs/user-guide/data-pipelines/index.md

```diff
@@ -0,0 +1,14 @@
+# Data Pipelines
+
+In machine learning, _data pipelines_ start with data inputs, then process them
+in one or more [stages](#stages) (producing intermediate artifacts), and finally
+output a model, along with its performance metadata.
+
+If you find yourself repeating sequence of actions to get update the results of
+your project, then you may already have a pipeline. For example, a typical ML
+workflow could involve:
+
+1. Gathering data and for training and validation (once)
+2. Extracting useful features from the training dataset
+3. (Re)training an ML model
+4. Evaluating the results against the validation set
```

Comment:
we need to have some continuation here, `DVC provides a way to describe these steps in a YAML file, so ...` - ... brief summary of the section, add herre

Reply:
I already added something like this recently, PTAL.

Reply:
hmm, I don't see these changes?

Reply:
> I don't see these changes

It reads ``DVC helps you make it more manageable and consistent by [defining it] in a standard format (`dvc.yaml`).`` but it's in the paragraph above, not after the list.

Reply:
ah, I see that confused me. Don't you think this way this list still requires some continuation?

Reply:
OK I reorganized the sentences for a better flow.